_includes/v22.2/start-in-docker/mac-linux-steps.md

```diff
@@ -1,38 +1,43 @@
-## Step 1. Create a bridge network
+### Step 1. Create a bridge network
 
 Since you'll be running multiple Docker containers on a single host, with one CockroachDB node per container, you need to create what Docker refers to as a [bridge network](https://docs.docker.com/engine/userguide/networking/#/a-bridge-network). The bridge network will enable the containers to communicate as a single cluster while keeping them isolated from external networks.
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ docker network create -d bridge roachnet
+docker network create -d bridge roachnet
 ~~~
 
 We've used `roachnet` as the network name here and in subsequent steps, but feel free to give your network any name you like.
 
-## Step 2. Start the cluster
+### Step 2: Create Docker volumes for each cluster node
 
-1. Create a [Docker volume](https://docs.docker.com/storage/volumes/) for each container:
+Cockroach Labs recommends that you store cluster data in Docker volumes rather than in the storage layer of the running container. Otherwise, if a Docker container is inadvertently deleted, its data is inaccessible.
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    docker volume create roach1
-    ~~~
+Create a [Docker volume](https://docs.docker.com/storage/volumes/) for each container:
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    docker volume create roach2
-    ~~~
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach1
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach2
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+docker volume create roach3
+~~~
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    docker volume create roach3
-    ~~~
+
+### Step 3. Start the cluster
```

Comment:
The formatting of some of the substeps are broken.
<img width="1077" alt="Screen Shot 2022-09-29 at 2 43 58 PM" src="https://user-images.githubusercontent.com/20482960/193147657-f235991a-b253-4f05-b058-9be8f5323703.png">

Reply:
Fixed.